spec/controllers/people_controller_spec.rb

```diff
@@ -99,8 +99,8 @@
         end
 
         it "assigns people" do
-          eugene2 = FactoryGirl.create(:person,
-                            :profile => FactoryGirl.build(:profile, :first_name => "Eugene",
+          eugene2 = FactoryBot.create(:person,
+                            :profile => FactoryBot.build(:profile, :first_name => "Eugene",
```

Comment:
- Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line.
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.